sympy/core/function.py

```diff
@@ -700,7 +700,14 @@ def _eval_nseries(self, x, n, logx):
                     raise PoleError("Cannot expand %s around 0" % (self))
                 series = term
                 fact = S.One
-                _x = Dummy('x')
```

Comment:
`_x = _uniquely_named_symbol('xi', self)` will work after `from sympy.core.symbol import _uniquely_named_symbol`

Reply:
Didn't know this function. Thanks!